src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.spec.ts

```diff
@@ -57,9 +82,9 @@ describe('InstructorCourseDetailsPageComponent', () => {
 
   it('should snap with a course with one co-owner and no students, and populated course student list', () => {
     const stats: any = {
```

Comment:
The fact that you know that `*Total` are not the right fields means that this should not be `any`. Need to add the right typing for this.